.travis.yml

```
@@ -73,15 +73,16 @@ jobs:
         - doit package_build $CHANS $PKG_TEST_PYTHON --test-group=unit
       script: doit package_upload --token=$ANACONDA_TOKEN $LABELS
 
-
     - &pip_pkg
-      env: PYPI=testpypi PYPIUSER=$TPPU PYPIPASS=$TPPP
+      <<: *conda_default
+      env: PYPI="https://test.pypi.org/legacy/" PYPIUSER=$TPPU PYPIPASS=$TPPP
```

Presumably reverting the shortcut was unintended side effect of copy/paste?

```suggestion
      env: PYPI=testpypi PYPIUSER=$TPPU PYPIPASS=$TPPP
```

I've never tried this 'suggestion' thing before...

---

No this was intentional, at some point the package upload started failing with some error about testpypi not being a valid argument.

---

[Here](https://travis-ci.org/pyviz/panel/jobs/442481612)'s the actual error

```
$ doit ecosystem=pip package_upload -u $PYPIUSER -p $PYPIPASS -pypi ${PYPI}
ERROR: Invalid parameter: "testpypi". Must be a command, task, or a target.
Type "doit help" to see available commands.
Type "doit list" to see available tasks.
```

---

Ah I see my error, will fix it now.